### PR TITLE
DOCSP-20368: Link updates for subdomain consolidation

### PR DESCRIPTION
**Pull request by sarah-olson-mongodb**

### CODE REVIEW

**Review by schmalliso — APPROVED** [review]

**Comment by schmalliso** [comment]
Looks great! One out-of-scope note is that conf-sitemap.xml doesn't actually do anything anymore, so you could go ahead and delete it (it was necessary for the legacy toolchain, but the new toolchain generates the sitemap while it builds the site).

File: snooty.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "mongodb-vscode"
 title = "MongoDB for VS Code"
 
-intersphinx = ["https://docs.mongodb.com/manual/objects.inv"]
+intersphinx = ["https://mongodb.com/docs/manual/objects.inv"]
```
Comment (schmalliso) [inline snooty.toml:4]:
```suggestion
intersphinx = ["https://www.mongodb.com/docs/manual/objects.inv"]
```

Comment (schmalliso) [reply snooty.toml:4]: Missed on the first pass - sorry!

Comment (sarah-olson-mongodb) [reply snooty.toml]: Done! Ty.

**Pull request merged**